http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200123_cluster_table_gce_refactor.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="qi" id="cluster_table_gce_refactor">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="RUNTIME_CONFIG"/>
+            </not>
+        </preConditions>
+
+        <dropNotNullConstraint columnName="masterMachineType" tableName="CLUSTER" columnDataType="varchar(256)"/>
+        <dropNotNullConstraint columnName="masterDiskSize" tableName="CLUSTER" columnDataType="INT"/>
+        <dropNotNullConstraint columnName="numberOfWorkers" tableName="CLUSTER" columnDataType="INT"/>
+
+<!--                  Create RUNTIME_CONFIG table   -->
+        <createTable tableName="RUNTIME_CONFIG">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="clusterId" type="BIGINT"></column>
+            <column name="cloudService" type="ENUM('DATAPROC', 'GCE')" defaultValue="DATAPROC">
+                <constraints nullable="false"/>
+            </column>
+
+            <column name="machineType" type="varchar(256)" defaultValue="n1-standard-4">
+                <constraints nullable="false"/>
+            </column>
+
+            <!-- This might change when we move to PD -->
+            <column name="diskSize" type="INT" defaultValue="500">
+                <constraints nullable="false"/>
+            </column>
+
+            <!-- dataproc only -->
+            <column name="numberOfWorkers" type="INT" defaultValue="0">
+                <constraints nullable="false"/>
+            </column>
+
+            <!-- dataproc only -->
+            <column name="workerMachineType" type="varchar(256)" defaultValue="NULL" />
+
+            <!-- dataproc only -->
+            <column name="workerDiskSize" type="INT" defaultValue="NULL" />
+
+            <!-- dataproc only -->
+            <column name="numberOfWorkerLocalSSDs" type="INT" defaultValue="NULL" />
+
+            <!-- dataproc only -->
+            <column name="numberOfPreemptibleWorkers" type="INT" defaultValue="NULL"/>
+
+            <column name="dateAccessed" type="TIMESTAMP(6)"/>
```

Comment:
Is this created date, or last modified date? I'm wondering if we need this since we already have those fields on CLUSTER.

Reply:
currently when we update runtimeConfig, we update `Cluster`'s dateAccess field. I guess we don't have to do that. Since we're supporting patch api, so I feel this field can be helpful separately in `RuntimeConfig` table.